aiida/calculations/diff/diff_calculation.py

```diff
@@ -0,0 +1,59 @@
+"""
+Calculations provided by aiida_diff tutorial plugin.
+
+Register calculations via the "aiida.calculations" entry point in the setup.json file.
+"""
+from aiida.common import datastructures
+from aiida.engine import CalcJob
+from aiida.orm import SinglefileData
+
+class DiffCalculation(CalcJob):
+    """
+    AiiDA calculation plugin wrapping the diff executable.
+
+    Simple AiiDA plugin wrapper for 'diffing' two files.
+    """
+    @classmethod
+    def define(cls, spec):
+        """Define inputs and outputs of the calculation."""
+        # yapf: disable
+        super(DiffCalculation, cls).define(spec)
+
+        spec.inputs['metadata']['options']['parser_name'].default = 'diff'
+        spec.inputs['metadata']['options']['resources'].default = {
+                'num_machines': 1,
+                'num_mpiprocs_per_machine': 1,
+                }
```

Comment:
is this not being autoformatted via the pre-commit hooks?
if it was, this would almost certainly change

Reply:
also, when modifying the code of the python scripts, please always check the corresponding text in the tutorial.
e.g. in this case the text expects these two lines to be after `spec.output`.